src/Indicator.vala

```diff
@@ -66,11 +74,28 @@ public class BluetoothIndicator.Indicator : Wingpanel.Indicator {
 
     public override void closed () {
     }
+
+    private void update_tooltip (bool state, bool paired) {
+        string description = _("Bluetooth is off");
+        string context = _("Middle-click to disable");
```

Comment:
I think it'll be better to have "Middle-click to disable Bluetooth" here instead of just "to disable". I personally think it'd be better to give a context, much like the other indicators where we now have "Middle-click to enable/disable DnD" (notifications) or "Middle-click to snooze" (nightlight).

Note that this does come with the setback of not looking super nice in languages that do not have an official wording for "Bluetooth". For instance, "disable Bluetooth" translates to "禁用 Bluetooth" which will have a mix of English and Mandarin so I'd suggest to wait for a response from the design team.

Reply:
I meant just line 80. Sorry, never done a review before.